apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -110,6 +123,41 @@ class LessonOverview extends Component {
     return options;
   };
 
+  renderPrintOptions = () => {
+    const pdfDropdownOptions = this.compilePdfDropdownOptions();
+
+    if (pdfDropdownOptions.length > 0 && currentLocale() === 'en-US') {
```

Comment:
Can `DefaultLocale` from `sharedConstants` be used in this context?
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/currentLocale.ts#L3